consensus/expected.go

```diff
@@ -83,7 +83,7 @@ type Processor interface {
 
 // TicketValidator validates that an input ticket is valid.
 type TicketValidator interface {
-	ValidateTicket(types.Ticket, types.Ticket, address.Address, uint64) (bool, error)
+	ValidateTicket(types.Ticket, types.Ticket, address.Address) (bool, error)
```

Comment:
This would violate the convention set in this file through the `Processor` interface so I am hesitant.  We really need a project wide convention so that we can do the right thing without thinking about it.

If you declare this convention here and now across the whole project I'll make this change, spread the word and open a tech debt issue for bringing code in line with this convention.